README.md

```diff
@@ -1,5 +1,7 @@
 # Mobile Gutenberg
 
+
+
```

Comment:
This is an artifact of my debugging which I missed before opening the PR `:shame:`

I'll remove it when the time comes to make new changes here.